ts/src/config.ts

```diff
@@ -115,16 +115,25 @@ export interface Config extends GoogleAuthOptions {
   // For testing with startLocal() only.
   localProfilingPeriodMillis?: number;
 
-
   // Debugging information for startLocal will be recorded every
   // localLogPeriodMillis milliseconds.
   // For testing with startLocal() only.
   localLogPeriodMillis?: number;
 
-
   // Duration of time profiles collected when using startLocal().
   // For testing with startLocal() only.
   localTimeDurationMillis?: number;
+
+  // When true, source map support will be disabled.
```

Comment:
nit: it would be good to explain what the user visible effects of disabling source maps support are.

Reply:
Done.